src/tiles/grid/item-tile.ts

```diff
@@ -35,6 +35,10 @@ export class ItemTile extends LitElement {
 
   render() {
     const itemTitle = this.model?.title;
+    const [viewCount, viewLabel] =
```

Comment:
This isn't _strictly_ part of the sort bar, but it fixes the grid tiles so they show the correct type of views depending on the selected sort option.